packages/theme-tasks/lib/tasks/sass.js

```diff
@@ -40,10 +38,17 @@ function build(glob) {
     let _glob = glob || paths.sass.src;
 
     return gulp.src(_glob)
-        .pipe(sass(sassOptions).on("error", sass.logError))
+        .pipe(sass(fastSassOptions).on("error", sass.logError))
+        .pipe(postcss(postcssPlugins))
+        .pipe(gulp.dest(paths.sass.dist));
+}
+function buildSlow(glob) {
```

Comment:
How about `buildAll` or similar? Slow or fast is relative :)

Reply:
It's actually really noticeable. fast takes about 3 to 5 seconds. slow takes about 30.

I will never the less think on new names

Reply:
applied changes